flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/AggregationCodeGenerator.scala

```diff
@@ -178,13 +294,15 @@ class AggregationCodeGenerator(
                |      ${aggMapping(i)},
                |      (${accTypes(i)}) accs.getField($i));""".stripMargin
           } else {
+            val setDataView = genDataViewFieldSetter(s"acc$i", i)
             j"""
                |    org.apache.flink.table.functions.AggregateFunction baseClass$i =
                |      (org.apache.flink.table.functions.AggregateFunction) ${aggs(i)};
-               |
+               |    ${accTypes(i)} acc$i = (${accTypes(i)}) accs.getField($i);
+               |    $setDataView
```

Comment:
directly call `genDataViewFieldSetter(s"acc$i", i)` here? 
Would make it easier to follow the `acc$i` variables.

Reply:
yes